.github/workflows/test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Install dependencies
         run: pnpm install --ignore-scripts
       - name: Run tests
-        run: pnpm test
+        run: pnpm test | grep -v "test:markdown"
```

Comment:
`grep -v` will just hide `test:markdown` output. But we will still spend CPU on Markdown linter and CI will still fails on Markdown issues (but without a proper output why).